configuration_manager/xml_parser.py

```diff
@@ -0,0 +1,107 @@
+# Copyright 2020 Forschungszentrum Jülich GmbH and Aix-Marseille Université
+# "Licensed to the Apache Software Foundation (ASF) under one or more
+#  contributor license agreements; and to You under the Apache License,
+#  Version 2.0."
+
+import xml.etree.ElementTree as ET
+
+
+class Parser(object):
```

Comment:
Python2 is deprecated. You can drop the object:
https://stackoverflow.com/questions/4015417/python-class-inherits-object